python/gps/algorithm/traj_opt/traj_opt_lqr_python.py

```diff
@@ -221,13 +239,20 @@ def backward(self, prev_traj_distr, traj_info, eta, algorithm, m):
 
                 # Add in the value function from the next time step.
                 if t < T - 1:
-                    #TODO: Should multiply by
-                    #      (pol_wt[t+1] + eta)/(pol_wt[t] + eta) here.
-                    Qtt = Qtt + \
-                            Fm[t, :, :].T.dot(Vxx[t+1, :, :]).dot(Fm[t, :, :])
-                    Qt = Qt + \
-                            Fm[t, :, :].T.dot(Vx[t+1, :] +
-                                              Vxx[t+1, :, :].dot(fv[t, :]))
+                    if type(algorithm) == AlgorithmBADMM:
```

Comment:
Can we just write this as something like:
if type(algorithm) == AlgortihmBADMM:
    multiplier = (pol_wt[t+1] + eta)/(pol_wt[t] + eta)
else:
    multiplier = 1
Qtt = Qtt + multiplier * ...

I think this would be easier to read, and slightly less code. If you can think of a better variable name for multiplier, feel free to change.